Prelude.xcodeproj/project.pbxproj

```diff
@@ -1152,7 +1152,6 @@
 				PRODUCT_NAME = Prelude_UIKit;
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
-				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```

Comment:
im removing this so that the target just uses the inherited values from the project